### PR TITLE
Digital in

### PR DESCRIPTION
**Pull request by sheimers**
I have tried to extend the adafruit_si4713.py module to enable the digital audio input (I2S) of the Si4721 chip. It's still compatible with analog in / Si4713.
I had limited success, I eventually got digital audio in working, but not reproducible.

In addition to the modified adafruit_si4713.py, this pull request also includes a simple example script for a transmitter with digital audio input, examples/simpleDigitalTx.py
The example includes comments that describe how the Si4721 can be connected to a Raspberry Pi with I2S (hifiBerry) audio.
I'd be glad if someone could check my modifications and tell if there is a mistake or if it could be improved.

### CODE REVIEW

**Comment by sheimers** [comment]
I did some more tests, it's not stable with the PCM output from the Raspi GPIO, but it works perfectly and reproducible with a CP2615 USB-to-I2S bridge evaluation board.

**Comment by askpatrickw** [comment]
@sheimers  did you have a change to look at the CI failures?

File: adafruit_si4713.py [hunk 1]
```diff
@@ -343,6 +363,46 @@ def tx_antenna_capacitance(self):
...
+
+    @digital_input_sample_rate.setter
+    def digital_input_sample_rate(self, val):
+        print("test DISR, with value")
+        assert (val == 0) or (32000 <= val <= 48000)
+        print("will set", val)
+        if self._set_property(_SI4713_PROP_DIGITAL_INPUT_SAMPLE_RATE, val):
+            print("have set, return true")
+        else:
+            print("have set, return false")
```
Comment (FoamyGuy) [inline adafruit_si4713.py:384]: same as other prints, suggest removing or adding a `verbose` argument to `__init__()` that enables the printing

File: adafruit_si4713.py [hunk 2]
```diff
@@ -343,6 +363,46 @@ def tx_antenna_capacitance(self):
...
+
+    @digital_input_format.setter
+    def digital_input_format(self, val):
+        print("test digital input format, with value")
+        # assert (val == 0) or (32000 <= val <= 48000)
+        print("will set", val)
+        if self._set_property(_SI4713_PROP_DIGITAL_INPUT_FORMAT, val):
+            print("have set, return true")
+        else:
+            print("have set, return false")
```
Comment (FoamyGuy) [inline adafruit_si4713.py:404]: A few more here, suggest removing prints or adding a `verbose` argument to init to enable them.

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
I added a few commits to get this passing the CI checks. 

I'm not super familiar with this hardware, and I don't have one for testing, I did review the code though and have one suggestion to either remove the various print statements or add a `verbose` argument to the constructor that enables them.

**Comment by FoamyGuy** [comment]
Going to close this for now. @sheimers if you get back to this to make the requested changes please free to re-open this, same for anyone else if someone else finds this and is interested in working on this functionality. You can also ping me if you have any trouble or need help.

**Pull request closed without merging**